schema.graphql

```diff
@@ -1,21 +1,77 @@
 type Query {
-    books: [Book]
     queryTest: String!
     queryRestaurants: [Restaurant]
     queryOrder(orderId:String): [Order]
     queryOrders: [Order]
     hello:String
+    items(restaurantID: ID!):[Item!]
+    todayOrders(restaurantID: ID!): [Order!]
+
 }
 type Mutation{
     createOrder(order:inputOrder): String
-    broadcastRandomNumber: Float
+    createItem(data: CreateItemInput!): Item! # 新增一道菜，要自動生成 id 給他
+    updateItem(id: String!, data: UpdateItemInput!): Item! # 更新某道菜的資訊
+    deleteItem(id: String!): Item! # 刪除某道菜
 }
 type Subscription{
-    order(id:String):[Order]
+    order(restautantID: ID!): OrderSubscriptionPayload! # 等待新的訂單進來
     globalCounter: Float
     hello:[String]
     numberIncremented: Int
 }
+#! why need MutationType
```

Comment:
what do you mean by this comment

Reply:
It's just used for discussing schema with frontend stuff. I remove it.